src/new-components/Tooltip.tsx

```diff
@@ -0,0 +1,106 @@
+import styled from '@emotion/styled'
+import { colorsV2 } from '@hedviginsurance/brand'
+import { Questionmark } from 'components/icons/Questionmark'
+import { motion } from 'framer-motion'
+import * as React from 'react'
+
+interface Props {
+  body: string
+  size?: 'sm' | 'lg'
```

Comment:
Let's define this thing as a type 😉 

Something like

```typescript
type Size = 'sm' | 'lg'

interface Sized {
  size?: Size
}
```

Reply:
Yes, much more neat 👌